packages/desktop/components/AccountManagementDetails.svelte

```diff
@@ -39,11 +46,32 @@
     $: isImplicitAccount = isImplicitAccountOutput(selectedOutput.output as CommonOutput)
     $: accountId = isAccountOutput(selectedOutput) ? (selectedOutput?.output as AccountOutput)?.accountId : null
     $: address = accountId ? getBech32AddressFromAddressTypes(new AccountAddress(accountId)) : null
+    $: hasStakingFeature = hasStakingFeature(selectedOutput)
 
     function onExplorerClick(): void {
         const url = `${explorerUrl}/${ExplorerEndpoint.Output}/${selectedOutput.outputId.toString()}`
         openUrlInBrowser(url)
     }
+
+    function hasStakingFeature(output: OutputData): boolean {
+        return (
+            isAccountOutput(output) &&
+            (output.output as AccountOutput).features?.some((feature) => feature.type === FeatureType.Staking)
+        )
+    }
+
+    function getStakedAmount(): string | undefined {
+        if (!hasStakingFeature) return
+        let amount = '0'
```

Comment:
Why amount is a string here, we pass it as string in `amount = stakingFeature?.stakedAmount?.toString()` but then we convert it to `Number` when we call `formatTokenAmountBestMatch`?